src/rashdf/geom.py

```diff
@@ -364,6 +371,56 @@ def refinement_regions(self) -> GeoDataFrame:
             crs=self.projection(),
         )
 
+    def structures(self) -> GeoDataFrame:
+        """Return the model structures.
+
+        Returns
+        -------
+        GeoDataFrame
+            A GeoDataFrame containing the model structures if they exist.
+        """
+        if "/Geometry/Structures" not in self:
+            return GeoDataFrame()
+        struct_data = self["/Geometry/Structures"]
+        v_conv_val = np.vectorize(convert_ras_hdf_value)
+        sd_attrs = struct_data["Attributes"][()]
+        struct_dict = {"struct_id": range(sd_attrs.shape[0])}
+        struct_dict.update(
+            {name: v_conv_val(sd_attrs[name]) for name in sd_attrs.dtype.names}
+        )
+        v_conv_dt = np.vectorize(convert_ras_datetime)
```

Comment:
Per comment on `utils.py`, I think we should leave `datetime` and `Timestamp` objects as is in the GeoDataFrame.

Reply:
added optional `datetime_to_str` argument within `structures` (default `False`) method to allow the user to easily handle this when trying to write the `GeoDataFrame` to an ESRI shapefile